content/docs/gto/user-guide/index.md

```diff
@@ -36,42 +26,32 @@ An example `artifacts.yaml` can be found
 
 You may need to get a specific artifact version to a certain environment, most
 likely the latest one or the one currently assigned to the stage. Use `gto show`
-to find the [Git reference] (tag) you need (note that
-[CI platforms](#acting-in-ci-cd) may expose it for you, e.g. the `GITHUB_REF`
-[env var in GitHub Actions]):
+to find the [Git reference] (tag) you need.
 
 [git reference]: https://git-scm.com/book/en/v2/Git-Internals-Git-References
-[env var in github actions]:
-  https://docs.github.com/en/actions/learn-github-actions/environment-variables
-
-<admon type="tip">
-
-GTO doesn't provide a way to deliver the artifacts, but you can [use DVC] or
-[employ MLEM] for that.
-
-[use dvc]: /doc/gto/user-guide/dvc
-[employ mlem]: https://mlem.ai
 
-</admon>
+Get the git tag for the latest version:
 
 ```cli
 $ gto show churn@latest --ref
 churn@v3.1.1
+```
+
+Get the git tag for the version in `prod` stage:
 
-$ gto show churn#prod --ref  # by assigned stage
+```cli
+$ gto show churn#prod --ref
 churn@v3.0.0
 ```
 
-You may need the artifact's file path. If
-[annotated](#annotations-in-artifactsyaml), it can be discovered with
-`gto describe`:
+To download artifact files tracked with DVC, you can use the `dvc get` or
+`dvc import` commands (or simply use `dvc pull` if you `cd` inside the repo).
 
 ```cli
-$ gto describe churn@v3.0.0 --path
-models/churn.pkl
+$ dvc get $REPO $ARTIFACT_PATH --rev $REVISION -o $OUTPUT_PATH
```

Comment:
Having the command here makes sense, but again I think we need to clarify what GTO does. I think it's fine to keep the old language that GTO doesn't provide a way to deliver the artifacts as long as we also provide this example.